examples/numpyarrays/examply.nim

```diff
@@ -0,0 +1,37 @@
+import nimpy
+import scinim/numpyarrays
+import std/[times, monotimes, math, sequtils]
+
+template timeIt(name:string, body) = 
+  let t0 = getMonoTime()
+  body
+  let sub = getMonoTime() - t0
+  # echo(name, " took ", $(sub))
+  let elapsed {.inject.} = sub.inMicroseconds() / 1000
+
+proc doStuff[T](el: T) : T {.inline.} =
+  result = (1.0-el)/(1.0+el)
+
+proc modArrayInPlace*(x: NumpyArray[float64]) : NumpyArray[float64] {.exportpy.} =
+  # echo "modArrayInPlace.nim"
+  # Example of accessing the buffer directly
+  var ux = toUnsafeView(x)
+  ux[0] = 123
+  let np = pyImport("numpy")
+  discard np.put(x, 1, -5)
```

Comment:
Is it a good idea to use a `pyImport` in such a proc in an example? If someone just blindly starts from the example they might end up doing that (for no reason) many, many times instead of importing once (possibly via some `initNim` like proc?). Note that I'm not quite sure how to best change the code for that either.

Reply:
This was an example of a Python -> Nim -> Python pipeline but you may be right